README.md

```diff
@@ -144,6 +145,11 @@ class ExampleComponent {
     @Input() defaultTileLayerLabel = 'Kaart';
     /* Custom leaflet tile layer, if provided, shows actions on the leaflet to toggle between default and custom tile layer. */
     @Input() tileLayer: LeafletTileLayerModel;
+    /**
+     * The zoom level when a marker is selected.
+     * If null the zoomlevel won't change after marker selection.
+     */
+    @Input() onMarkerSelectZoom? = 16;
```

Comment:
ik vind het verwarrend dat dit begint met "on". misschien hernoemen naar "zoomOnMarkerSelect"?

Reply:
"zoomOnMarkerSelect" is een betere benaming, heb dit aangepast